ex20_ch4.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state constants
+EC_RES_TAX = 0.005
+DUNN_RES_TAX = 0.004
+IL_SALES_TAX = 8 
+
+#step1: get input from user
+order_amount = float(input("Enter the order amount:$"))
+state = input("Enter the state you reside in. Use abbreviations (eg, OH or MA): ")
+county = input("Enter the county you live in: ")
+
+#step2: set up first conditional
+if "WI" in state or "wi" in state: 
+    if county == "Eau Clair" or county == "eau clair":
```

Comment:
Lines 22 -28 are perfect. Checking the first county, then the second and if neither apply, then using the `else` condition. Nice job! 👏